purchase_line_product_image/models/__init__.py

```diff
@@ -0,0 +1,6 @@
+# -*- coding: utf-8 -*-
+# © 2017
+#   @Lucky Kurniawan <kurniawanluckyy@gmail.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from . import purchase_order
```

Comment:
[Add a carriage return at the end of this line](https://robots.thoughtbot.com/no-newline-at-end-of-file)